coding-concepts.js

```diff
@@ -15,40 +15,40 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 9
+// b) Verify and explain: wrong i thought it was talking about index so i started counting at 0
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+ console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: o
+// b) Verify and explain: i was correct it logged index 4, H is 0 and it went from there
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+ console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: i was right it logged ruby which was in inde 1 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+ //console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will log everything to upper case 
+// b) Verify and explain: it didnt logged everything to uppercase 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean", "undefined"]
-// console.log(typeof dataTypes.length)
+ console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will the length of each string
+// b) Verify and explain: it logged number, im not sure why
```

Comment:
`.length` is being used on the `dataTypes` variable, which will return 4 because there are 4 elements in the array. The `typeof` operator returns what the data type of the data that follows it is. Since `dataTypes.length` is returning 4, typeof is looking at 4 and seeing that it is a number, therefore it returns number.